docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -82,214 +68,347 @@ To enable your application to use the image util functionality:
+   ret = image_util_transform_destroy(transform_h);
+   ```
+
+<a name="rotate_mediapacket"></a>
+## Rotating Media Packet
+
+To rotate a media packet:
+
+1. Create a transformation handle using `image_util_transform_create()`:
+
    ```
-<a name="crop"></a>
-## Cropping an Image
+   transformation_h transform_h;
+   ret = image_util_transform_create(&transform_h);
+   ```
+
+2. Set the amount of rotation using `image_util_transform_set_rotation()`:
+
+   ```
+   ret = image_util_transform_set_rotation(transform_h, rotation);
+   ```
+
+    The possible values for the `rotation` parameter are defined in the `image_util_rotation_e` enumeration (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__IMAGE__UTIL__TRANSFORM__MODULE.html#gad0682da8519f229944c9c5617b7a1107) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__IMAGE__UTIL__TRANSFORM__MODULE.html#gad0682da8519f229944c9c5617b7a1107) applications).
+
+3. Execute the transformation using `image_util_transform_run()`:
+
+   ```
+   ret = image_util_transform_run(transform_h, (media_packet_h)src_packet,
+                                (image_util_transform_completed_cb)completed_callback,
+                                user_data);
+   ```
+
+   > **Note**
+   >
+   > The image format has no effect on the transformation. If the color space is YUV, then the width and height of the target image must be multiples of eight. This restriction does not apply to the RGB images.
+
+4. Handle the transformation results in `image_util_transform_completed_cb()`, which is invoked after the transformation is complete.
+
+5. After the transformation is complete, destroy the transformation handle using `image_util_transform_destroy()`:
+
+   ```
+   ret = image_util_transform_destroy(transform_h);
+   ```
+
+<a name="crop_image"></a>
+## Cropping Image
 
 To crop an image:
 
-1. Create a transformation handle using the `image_util_transform_create()` function:
+1. Create a transformation handle using `image_util_transform_create()`:
 
    ```
-   transformation_h handle;
-   ret = image_util_transform_create(&handle);
+   transformation_h transform_h;
+   ret = image_util_transform_create(&transform_h);
    ```
 
-2. Set the crop area using the `image_util_transform_set_crop_area()` function:
+2. Set the crop area using `image_util_transform_set_crop_area()`:
 
    ```
-   ret = image_util_transform_set_crop_area(handle, start_x, start_y, end_x, end_y);
+   ret = image_util_transform_set_crop_area(transform_h, start_x, start_y, end_x, end_y);
    ```
 
-3. Execute the transformation using the `image_util_transform_run()` function:
+3. Execute the transformation using `image_util_transform_run()`:
 
    ```
-   ret = image_util_transform_run(handle, src,
-                                  (image_util_transform_completed_cb)completed_callback,
-                                  user_data);
+   ret = image_util_transform_run2_async(transform_h, src_image,
```

Comment:
`image_util_transform_run()` in the description, `image_util_transform_run2_async` in the sample code.

I think `image_util_transform_run2_async` would be correct as the functions with 2 in the name are used for Image. However, would you please check?

Reply:
`image_util_transform_run()` in the description will be replaced to `mage_util_transform_run2()` or `mage_util_transform_run2_async()`.